Makefile.am

```diff
@@ -57,9 +71,10 @@ all-local:
 
 dist-hook: gitrev
 	cp ${top_srcdir}/include/zfs_gitrev.h $(distdir)/include; \
-	sed -i 's/Release:[[:print:]]*/Release:      $(RELEASE)/' \
+	sed -i.bak -e 's/Release:[[:print:]]*/Release:      $(RELEASE)/' \
```

Comment:
Why is a backup required here?

Reply:
[it is required](https://www.freebsd.org/cgi/man.cgi?query=sed&sektion=&n=1).

Reply:
Similar to my other comment, would `sed -i''` work here for FreeBSD?

Reply:
Should this be `sed ${ac_inplace}` for consistency?

Reply:
Yes I'll fix that, thanks.